website/docs/r/oracle_autonomous_database_regular.html.markdown

```diff
@@ -0,0 +1,112 @@
+
+* `data_storage_size_in_tbs` - (Required) The maximum storage that can be allocated for the database, in terabytes.
+
+* `db_version` - (Required) A valid Oracle Database version for Autonomous Database.
+
+* `db_workload` - (Required) The Autonomous Database workload type. The following values are valid:
+    * OLTP - indicates an Autonomous Transaction Processing database
+    * DW - indicates an Autonomous Data Warehouse database
+    * AJD - indicates an Autonomous JSON Database
+    * APEX - indicates an Autonomous Database with the Oracle APEX Application Development workload type.
+
+* `display_name` - (Required) The user-friendly name for the Autonomous Database. The name does not have to be unique.
+
+* `auto_scaling_enabled` - (Required) Indicates if auto scaling is enabled for the Autonomous Database CPU core count. The default value is `true`.
+
+* `auto_scaling_for_storage_enabled` - (Required) Indicates if auto scaling is enabled for the Autonomous Database storage. The default value is `false`.
+
+* `mtls_connection_required` - (Required) Specifies if the Autonomous Database requires mTLS connections.
+
+* `license_model` - (Required) The Oracle license model that applies to the Oracle Autonomous Database. Bring your own license (BYOL) allows you to apply your current on-premises Oracle software licenses to equivalent, highly automated Oracle services in the cloud. License Included allows you to subscribe to new Oracle Database software licenses and the Oracle Database service. Note that when provisioning an [Autonomous Database on dedicated Exadata infrastructure](https://docs.oracle.com/en/cloud/paas/autonomous-database/index.html), this attribute must be null. It is already set at the Autonomous Exadata Infrastructure level. When provisioning an [Autonomous Database Serverless] (https://docs.oracle.com/en/cloud/paas/autonomous-database/index.html) database, if a value is not specified, the system defaults the value to `BRING_YOUR_OWN_LICENSE`. Bring your own license (BYOL) also allows you to select the DB edition using the optional parameter.
+
+* `location` - (Required) The Azure Region where the Autonomous Databases should exist. Changing this forces a new Autonomous Databases to be created.
+
+* `name` - (Required) The name which should be used for this Autonomous Databases.
+
+* `national_character_set` - (Required) The national character set for the autonomous database.  The default is AL16UTF16. Allowed values are: AL16UTF16 or UTF8.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Autonomous Databases should exist.
+
+* `subnet_id` - (Required) The [OCID](https://docs.cloud.oracle.com/iaas/Content/General/Concepts/identifiers.htm) of the subnet the resource is associated with.
+
+* `vnet_id` - (Required) The ID of the vnet associated with the cloud VM cluster.
```

Comment:
Arguments in the resource documentation should be ordered in the following way:

1. name, resource_group_name, location
2. all other required properties in alphabetical order
3. all other optional properties in alphabetical order

Reply:
Done, thank you.